packages/shell-api/src/enums.ts

```diff
@@ -15,6 +15,11 @@ export const ALL_SERVER_VERSIONS = [ ServerVersions.earliest, ServerVersions.lat
 export const ALL_TOPOLOGIES = [ Topologies.ReplSet, Topologies.Sharded, Topologies.Standalone ];
 export const ALL_PLATFORMS = [ ReplPlatform.Compass, ReplPlatform.Browser, ReplPlatform.CLI ];
 
+export const toStringMethods = {};
+toStringMethods[ReplPlatform.Compass] = 'TODO';
```

Comment:
What should these be for Compass/Browser?

Reply:
Ok, will do! Would that work the same for all the platforms?

Reply:
I think so, actually `require('util').inspect.custom` should always work.